arm-relay/2016-07-01/swagger/relay.json

```diff
@@ -1951,7 +1945,7 @@
           ],
           "x-ms-enum": {
             "name": "policyKey",
-            "modelAsString": true
+            "modelAsString": false
```

Comment:
are you intentionally changing this property to a enum that can only be the values you indicated above?https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum